src/dotnet-restore/project.json

```diff
@@ -20,7 +20,6 @@
     "NuGet.Packaging.Core.Types": "3.4.0-beta-515",
     "NuGet.Versioning": "3.4.0-beta-515",
     "Microsoft.NETCore.Platforms": "1.0.1-rc2-23704",
-    "Microsoft.NETCore.TestHost": "1.0.0-rc2-23704",
```

Comment:
This for sure broke restore. Latest rel is using corehost now but this branch is not yet.